examples/rust/opensk/src/lib.rs

```diff
@@ -64,12 +68,26 @@ struct WasefireEnv {
     // c: PhantomData<C>,
 }
 
-impl AttestationStore for WasefireEnv {
-    fn get(&mut self, id: &Id) -> Result<Option<Attestation>, Error> {
-        todo!()
+impl Persist for WasefireEnv {
+    fn find(&self, key: usize) -> CtapResult<Option<Vec<u8>>> {
+        // TODO: Convert `Option<Vec<u8>>` to `Option<Box<[u8]>>`
+        Ok(store::find(key)?)
     }
 
-    fn set(&mut self, id: &Id, attestation: Option<&Attestation>) -> Result<(), Error> {
+    fn insert(&mut self, key: usize, value: &[u8]) -> CtapResult<()> {
+        // TODO: Impl `From<wasefire::Error>` for `Ctap2StatusCode`
+        Ok(store::insert(key, value)?)
+    }
+
+    fn remove(&mut self, key: usize) -> CtapResult<()> {
+        Ok(store::remove(key)?)
+    }
+
+    fn iter(&self) -> CtapResult<PersistIter<'_>> {
+        // Ok(Box::new(self.store.iter()?.map(|handle| match handle {
+        //     Ok(handle) => Ok(handle.get_key()),
+        //     Err(error) => Err(error.into()),
+        // })))
```

Comment:
For this we need to extend the applet store API. I'll take a look at how to do it, because either we use allocation or we do many syscalls. I expect the first to be better, but need to take a look.

Reply:
Done.